.github/workflows/build.yml

```diff
@@ -19,8 +19,15 @@ jobs:
       matrix:
         os:
           - windows-2019
+          - ubuntu-latest
+          - macos-latest
         node:
           - 10
+          - 12
+          - 14
+          - 16
+          # - 18
+          # - 20
```

Comment:
Please add a comment with an issue number to help readers understand what is blocking these versions.

Reply:
Sure. I was planning on merging #141 before this one, just waiting on someone to double check it, but I don't have to.